static/app/views/insights/common/components/tableCells/starredTransactionCell.tsx

```diff
@@ -0,0 +1,77 @@
+import {useState} from 'react';
+
+import {Button} from 'sentry/components/core/button';
+import {IconStar} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {FlexContainer} from 'sentry/utils/discover/styles';
+import useProjects from 'sentry/utils/useProjects';
+import {
+  type StarTransactionParams,
+  useStarredTransactions,
+} from 'sentry/views/insights/common/utils/useStarredTransactions';
+
+interface Props {
+  containerProps?: React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  >;
+  isStarred?: boolean;
+  projectSlug?: string;
+  transactionName?: string;
```

Comment:
Why `transactionName` here, but `segmentName` in the hook?

Reply:
In the code it should be the same everywhere if possible! We can update or change labels at any point

Reply:
updated to be `segment` everywhere